nibabel/tests/test_arrayproxy.py

```diff
@@ -50,11 +50,6 @@ def copy(self):
         return FunkyHeader(self.shape)
 
 
-class CArrayProxy(ArrayProxy):
```

Comment:
(I think based on your suggestion) I included the CArrayProxy class to arrayproxy.py because I need it for BV file formats. As this class was defined in test_arrayproxy.py as well I rather imported it and deleted the duplicate. But maybe I did not understand the logic here (or your question :)).